### ISSUE TITLE
minizip's iowin32 does not properly detect Windows Store / UWP

### ISSUE THREAD

**Issue by null77** [madler/zlib#480]
Currently iowin32 uses

```
// see Include/shared/winapifamily.h in the Windows Kit
#if defined(WINAPI_FAMILY_PARTITION) && (!(defined(IOWIN32_USING_WINRT_API)))
#if WINAPI_FAMILY_ONE_PARTITION(WINAPI_FAMILY, WINAPI_PARTITION_APP)
#define IOWIN32_USING_WINRT_API 1
#endif
#endif
```

However this code no longer works correctly to detect Windows Store/UWP. This should be updated to something like the following:

```
#if !defined(IOWIN32_USING_WINRT_API)
#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY == WINAPI_FAMILY_APP)
#define IOWIN32_USING_WINRT_API 1
#endif
#endif
```

**Comment by gabriellanzer** [comment]
I was having issues compiling ZLib with Conan on my Clang11+Win10 setup. The `WINAPI_FAMILY_ONE_PARTITION` was undefined, changing for your code fixed my issue. :D Thanks.

**Comment by tbeu** [comment]
Seems to be addressed by ce12773790517034317274f5c65ba70cfeea29f7.

**Comment by gvollant** [comment]
see https://github.com/madler/zlib/pull/610
